mappings/cwe.schema.json

```diff
@@ -11,7 +11,7 @@
       "required": ["default"]
     },
     "VRTid": { "type": "string", "pattern": "^[a-z_][a-z_0-9]*$" },
-    "CWE": { "type" : "array",
+    "CWE": { "type" : [ "array", "null" ],
```

Comment:
I think the current pattern is to simply not include the entry which is why we didn't need to allow for null here.